test/Microsoft.ML.Benchmarks/BigramAndTrigramBenchMark.cs

```diff
@@ -0,0 +1,82 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using BenchmarkDotNet.Attributes;
+using Microsoft.ML.Runtime;
+using Microsoft.ML.Runtime.Data;
+using Microsoft.ML.Runtime.RunTests;
+using Microsoft.ML.Runtime.Tools;
+using System.IO;
+using System.Text;
+
+namespace Microsoft.ML.Benchmarks
+{
+    // Adding this class to not print anything to the console.
+    // This is required for the current version of BenchmarkDotNet
+    internal class EmptyWriter : TextWriter
+    {
+        internal static readonly EmptyWriter Instance = new EmptyWriter();
+        public override Encoding Encoding => null;
+    }
+
+    public class BigramAndTrigramBenchmark
+    {
+        private static string s_dataPath_Wiki;
+        private static string s_modelPath_Wiki;
+
+        [GlobalSetup(Targets = new string[] { nameof(Preceptron_CV), nameof(LightGBM_CV) })]
```

Comment:
This setup needs to run for all the tests, right?  It initializes the `s_dataPath_Wiki` variable, which appears to be needed even in the `WikiDetox` benchmark.

Reply:
@eerhardt only one globalSetup can be applied to one target

Reply:
side note: please do remember that BenchmarkDotNet runs all these benchmarks in dedicated processes, so if `BenchmarkA` has `SetupMethodA` and it initializes a static field, another benchmark from the same class won't have the static field initialized and the setup needs to initialize it as well

This is why when I was doing a cleanup I made all of the static fields instance fields. So good practice is to avoid using static for fields to avoid confusion.